### PR TITLE
fix(frontend/backend): Add Friend page tm-40

### PR DESCRIPTION
**Pull request by GEOFARL**
- Updated the "Courses" title to be 20px
- Made the "Back" button to be as on the "Chat" and "Course details" pages and change after hovering
- Made the Follow" and "Following" buttons not to change regarding the user's name length (size should be 120px 32px)
- Made the GET /user-courses/{userId} response body not to be empty even when user has courses

### CODE REVIEW

File: apps/frontend/src/pages/user/styles.module.css [hunk 1]
```diff
@@ -47,15 +47,18 @@
...
 .return-button {
 	align-self: flex-start;
-	padding: 0;
+	width: 25px;
+	height: 25px;
 	margin-bottom: 17px;
 	color: var(--color-text-200);
-	background-color: transparent;
-	border-color: var(--color-background);
+	background-color: #e0e0e0;
```
Comment (Vfivew) [inline apps/frontend/src/pages/user/styles.module.css:54]: Do we need new varieable for it maybe?

**Review by Sytnikovakatya — APPROVED** [review]

File: apps/frontend/src/pages/user/styles.module.css [hunk 2]
```diff
@@ -47,15 +47,18 @@
...
-.return-button:hover {
-	color: var(--color-text-200);
-	background-color: transparent;
-	border-color: var(--color-background);
+.courses-title {
+	font-size: var(--font-size-400);
+}
+
+.follow-button {
+	width: 120px;
```
Comment (what1s1ove) [inline apps/frontend/src/pages/user/styles.module.css:63]:
```suggestion
	min-width: 120px;
```
never add a fix width

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/user/styles.module.css (hunk 1 above)
Comment (GEOFARL) [reply apps/frontend/src/pages/user/styles.module.css:54]: I do not think that it is necessary

Comment (KrasKa2) [reply apps/frontend/src/pages/user/styles.module.css:54]: @GEOFARL  Why do you think so? We have a rule, that everything should be the same. So if we use vars for color, we should use it everywhere

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**